source/manual/github-unavailable.html.md

```diff
@@ -56,24 +54,32 @@ In this scenario, Jenkins security should be disabled to enable deployment:
 
 1. SSH to the Jenkins deploy instance:
 
-```console
-gds govuk connect -e production ssh aws/jenkins
-```
+    ```console
+    # 2ndline drill: use the `integration/staging` rather than `production` environment in the following console.
+    gds govuk connect -e production ssh aws/jenkins
+    ```
 
 2. Disable Puppet: `govuk_puppet -r "Emergency Jenkins deploy" --disable`
-3. Edit the Jenkins configuration file: `sudo vim /var/lib/jenkins/config.xml`
-4. Replace `<useSecurity>true</useSecurity>` with `<useSecurity>false</useSecurity>` and save
-5. Restart Jenkins: `sudo service jenkins restart`
+
+2. Edit the Jenkins configuration file: `sudo vim /var/lib/jenkins/config.xml`
+
+2. Replace `<useSecurity>true</useSecurity>` with `<useSecurity>false</useSecurity>` and save
+
+2. Restart Jenkins: `sudo service jenkins restart`
 
 Note that once security is disabled, anyone on GDS trusted IPs will be able to deploy to that environment. This will bypass protection for Production - do not leave Production without security for any longer than necessary.
 
 See the [Jenkins documentation](https://jenkins.io/doc/book/system-administration/security/#disabling-security) for further details.
 
 ### Deploying the code change
 
-1. Review the pull request on AWS CodeCommit through the [AWS Console](https://eu-west-2.console.aws.amazon.com/codesuite/codecommit/repositories?region=eu-west-2) (access to GOV.UK repos must be granted by a GDS AWS administrator).
+1. `2ndline drill only:` Ensure that you check the `Freeze deployments?` box on the settings page of the app in the Release website.
```

Comment:
I'd refer to the "release app" rather than website, and a link to it might be helpful here.